dev-guide/ci/events-files.md

```diff
@@ -0,0 +1,66 @@
+# Openshift CI produced files related to events
+
+To aid in debugging, the Openshift CI (prow) produces various files related to events.  The files
+reside in the gather-must-gather and gather-extra subdirectories in the artifacts storage area.
+You will see the "Artifacts" link in the upper right of a prow job (for example, see
+[this prow job](https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-openshift-release-master-ci-4.11-upgrade-from-stable-4.10-e2e-gcp-ovn-upgrade/1505747388038385664).
```

Comment:
nit: we eventually garbage-collect these artifacts.  Might be worth explaining how to get to a recent run, like https://amd64.ocp.releases.ci.openshift.org/#4-stable -4.y.z-> https://amd64.ocp.releases.ci.openshift.org/releasestream/4-stable/release/4.10.12 -upgrade-> https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/release-openshift-origin-installer-e2e-aws-upgrade/1519313665751257088 ?